.github/workflows/linux.yml

```diff
@@ -698,12 +698,3 @@ jobs:
           else
             make V=1 pytest-ci
           fi
-
-      - name: 'build examples'
-        if: ${{ matrix.build.make-custom-target != 'tidy' }}
-        run: |
-          if [ -n '${{ matrix.build.generate }}' ]; then
-            ${{ matrix.build.make-prefix }} cmake --build . --verbose --target curl-examples
-          else
-            ${{ matrix.build.make-prefix }} make V=1 examples
-          fi
```

Comment:
Why drop building examples on Linux?

Reply:
oh, my mistake!

Reply:
Thanks for noticing my mistakes. Pushed an update.

Reply:
No worries, it seemed unintented indeed.